model/src/main/java/org/projectnessie/api/http/HttpNamespaceApi.java

```diff
@@ -65,7 +68,8 @@ public interface HttpNamespaceApi extends NamespaceApi {
     @APIResponse(responseCode = "404", description = "Reference not found"),
     @APIResponse(responseCode = "409", description = "Namespace already exists"),
   })
-  Namespace createNamespace(@BeanParam @NotNull NamespaceParams params)
+  Namespace createNamespace(
+      @BeanParam @NotNull NamespaceParams params, @NotNull @RequestBody NamespaceUpdate properties)
```

Comment:
If you change this to
```java
  Namespace createNamespace(
      @BeanParam @NotNull NamespaceParams params, @NotNull @RequestBody Namespace namespace) {}
```
you don't have to add the 
```java
@JsonIgnoreProperties(ignoreUnknown = true)
```
to `NamespaceUpdate` - nothing in `NamespaceUpdate` that's not in `Namespace`.
Also prevents compatibility headaches.
Feels a bit odd to have `Namespace` as a path-param and request-body - but yea - it's been that way anyways.

Reply:
yeah indeed that feels odd but would most likely work